route_test.go

```diff
@@ -2019,3 +2019,70 @@ func TestRouteFWMarkOption(t *testing.T) {
 		t.Fatal(routes)
 	}
 }
+
+func TestRouteGetFIBMatchOption(t *testing.T) {
+	tearDown := setUpNetlinkTest(t)
+	defer tearDown()
+
+	err := LinkAdd(&Dummy{LinkAttrs{Name: "eth0"}})
+	if err != nil {
+		t.Fatal(err)
+	}
+	link, err := LinkByName("eth0")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if err = LinkSetUp(link); err != nil {
+		t.Fatal(err)
+	}
+	addr := &Addr{
+		IPNet: &net.IPNet{
+			IP:   net.IPv4(192, 168, 0, 2),
+			Mask: net.CIDRMask(24, 32),
+		},
+	}
+	if err = AddrAdd(link, addr); err != nil {
+		t.Fatal(err)
+	}
+
+	// if err := RouteAdd(&Route{
+	// 	LinkIndex: link.Attrs().Index,
+	// 	Gw:        net.IPv4(192, 168, 0, 1),
+	// 	Dst:       nil,
+	// }); err != nil {
+	// 	t.Fatal(err)
+	// }
```

Comment:
Is this still needed? Otherwise it's probably better to delete the code.

Reply:
Woops, I just forgot to delete this. Fixed in the latest push.